tensornetwork/network.py

```diff
@@ -1300,3 +1323,27 @@ def __contains__(self, item):
       raise TypeError("Type '{}' was unexpected. "
                       "Only 'None' and 'Edge' types are allowed.".format(
                           type(item)))
+
+
+def load(path: str):
```

Comment:
Throw this in a different file since `network.py` is getting really large.

Reply:
Can put it in __init__ or do you prefer a new file?

Reply:
New file

Reply:
Moved to utils.py